whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapQueryHandler.java

```diff
@@ -0,0 +1,107 @@
+package net.ripe.db.whois.api.rdap;
+
+import com.google.common.collect.Lists;
+import com.google.common.net.InetAddresses;
+import net.ripe.db.whois.api.fulltextsearch.FullTextIndex;
+import net.ripe.db.whois.api.rest.ApiResponseHandler;
+import net.ripe.db.whois.common.dao.RpslObjectDao;
+import net.ripe.db.whois.common.domain.ResponseObject;
+import net.ripe.db.whois.common.rpsl.ObjectType;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import net.ripe.db.whois.common.source.SourceContext;
+import net.ripe.db.whois.query.acl.AccessControlListManager;
+import net.ripe.db.whois.query.domain.QueryCompletionInfo;
+import net.ripe.db.whois.query.domain.QueryException;
+import net.ripe.db.whois.query.handler.QueryHandler;
+import net.ripe.db.whois.query.planner.AbuseCFinder;
+import net.ripe.db.whois.query.query.Query;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+import java.net.InetAddress;
+import java.util.List;
+import static net.ripe.db.whois.common.rpsl.ObjectType.*;
+
+@Component
+public class RdapQueryHandler {
+
+    private static final int STATUS_TOO_MANY_REQUESTS = 429;
+    private static final Logger LOGGER = LoggerFactory.getLogger(RdapQueryHandler.class);
```

Comment:
Move the LOGGER field up so it's first (it's a generic field in a lot of classes, and by convention we separate it at the top).